src/App.css

```diff
@@ -0,0 +1,33 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
Since you've scraped off the basic react home page with the spinning logo, wouldn't it also make sense to remove some of the CSS classes? 

You don't really need .App-logo, App-header, App-link.

The only one you're using is `.App`

Reply:
Yes @njeri-ngigi . You are quite right. Let me work on this feedback to remove the extra CSS classes